apps/twap/src/app/routes/_chainId/_walletAddress/orders/index.ts

```diff
@@ -0,0 +1,185 @@
+import { FastifyInstance } from 'fastify';
+import { Wallet } from '../../../../data/wallet';
+import { Order } from '../../../../data/order';
+import { FromSchema, JSONSchema } from 'json-schema-to-ts';
+
+const ADDRESS_LENGTH = 42;
+
+const routeSchema = {
+  type: 'object',
+  required: ['chainId', 'walletAddress'],
+  properties: {
+    chainId: {
+      enum: ['1', '5', '100'],
```

Comment:
Are the networks used anywhere else?
Would it make sense to come from a central config location, or even env vars?

Reply:
In principle I agree, but would prefer to circle back on this to avoid premature abstraction. There are a couple of questions to be answered here since we cannot treat all the chains the same.

Env vars won't work in general though, as we are even defining the schema as _const_ to be inlined in build step. So probably still somewhere in code.